tests/test_benchmark/expected-results/benchmark.tsv

```diff
@@ -0,0 +1,6 @@
+jobid	rule	wildcards	resources	input_size_mb	threads	s	h:m:s	max_rss	max_vms	max_uss	max_pss	io_in	io_out	mean_load	cpu_time	input_file_size_mb
```

Comment:
I'd maybe also not put `:` in column names and call them: `cpu_time` and `wall_time` or something like this. Units can be looked up in the documentation, no need to state them in the column name.

Reply:
I see, `s` and `h:m:s` are just two ways of formatting the same number. Bit redundant but potentially worth it for ease of viewing. In that case, call it `wall_s` and `wall_hms` or something like that?

Reply:
pretty sure `h:m:s` is what has been used in current benchmark output so keeping this here is mostly for backward compatibility -- if an update is made after an incomplete run, changing this column name would be a breaking change.